opencti-platform/opencti-graphql/src/modules/indicator/indicator-domain.ts

```diff
@@ -305,6 +312,18 @@ export const indicatorEditField = async (context: AuthContext, user: AuthUser, i
   if (!indicator) {
     throw FunctionalError('Cannot edit the field, Indicator cannot be found.');
   }
+
+  const foundPattern = finalInput.find((item) => item.key === 'pattern');
+  if (!indicator.x_opencti_observables_values || foundPattern) {
```

Comment:
I also suggest to move this part after the validation check about valid_until, since if we don't pass this validation check, there is no need for this part.